docs/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial.md

```diff
@@ -212,3 +212,5 @@ ros2 bag play <ROSBAG_NAME>
 If the expected ROS topic data is not produced, or the point cloud is not visible on the simulation screen, report the problem as an [Autoware Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/).
 
 To make analysis of the problem easier, make sure to provide a description of your environment, a detailed set of reproduction steps, screenshots or a short video of any unexpected behaviour and attach the `Player.log` file, which can be found in the `~/.config/unity3d/Tier\ IV/E2ESimulator/Player.log` directory.
+
+[simple running video link](https://drive.google.com/file/d/1qxImNdK7d1zhMsJqhOa2NEM3vnjKSY7B/view?usp=share_link)
```

Comment:
```suggestion
![type:video](https://drive.google.com/file/d/1qxImNdK7d1zhMsJqhOa2NEM3vnjKSY7B/view?usp=share_link)
```
Following https://pypi.org/project/mkdocs-video/

cc @kenji-miyake do we have this plugin?

Reply:
Created a PR in https://github.com/autowarefoundation/autoware-github-actions/pull/202 @kenji-miyake

Reply:
Seeing https://github.com/squidfunk/mkdocs-material/issues/4053, it's may be possible to the HTML video tag.
@xmfcx Which do you think is better?